Hublsoft.Net.LoginLogout.Bll/UserManager.cs

```diff
@@ -42,6 +42,14 @@ public async Task<Guid> AuthenticateUserAsync(string emailAddress, string passwo
                 tasks.Add(_registeredUsersRepo.IncrementFailedLoginAttemptsAsync(id));
                 tasks.Add(_registeredUserAuditsRepo.AddFailedLoginAttemptAuditRecordAsync(id));
                 await Task.WhenAll(tasks);
+
+                var loginAttempts = await _registeredUsersRepo.GetFailedLoginAttempsAsync(id);
+
+                if (loginAttempts == 3)
+                {
+                    await _registeredUsersRepo.LockAccount(id);
+                }
```

Comment:
The third requirement to add an audit record that states the account is locked out, would need to be added here.